src/libcadet/model/parts/MultiChannelConvectionDispersionOperator.cpp

```diff
@@ -21,6 +21,8 @@
 #include "linalg/CompressedSparseMatrix.hpp"
 #include "model/parts/AxialConvectionDispersionKernel.hpp"
 #include "model/ParameterDependence.hpp"
+#include "model/exchange/LinearExchange.cpp"
```

Comment:
the abstract class should be included, not all exchange modules separately.
Anyways, it seems like its not even needed so please delete